app/shared/components/blocks/fund-summary-header/FundSummaryHeader.styles.ts

```diff
@@ -0,0 +1,98 @@
+import { mainHexPallete } from '~/components/design-system/all-components/theme/colors';
+
+export const TITLE_GRID_COLUMN = { xs: '1 / -1', sm: '1 / -1', md: '1 / -1' } as const;
+export const TITLE_SX = {
+  '& h2': {
+    fontSize: { xs: '24px', sm: '40px', md: '56px' },
+    maxWidth: { xs: '80%', sm: 'none' }
+  }
+} as const;
+
+export const styles = {
+  container: {
+    display: 'grid',
+    gridTemplateColumns: 'subgrid',
+    gridColumn: '1 / -1'
+  },
+
+  backLink: {
+    gridColumn: {
+      xs: '1 / -1',
+      sm: '1 / -1',
+      md: '1 / 6'
+    },
+    mb: '24px',
+    justifySelf: 'start',
+    alignSelf: 'start',
+    '& .MuiButton-root': {
+      px: 0
+    }
+  },
+
+  title: {
+    gridColumn: {
+      xs: '1 / -1',
+      sm: '1 / -1',
+      md: '1 / -1'
```

Comment:
You don't need to duplicate values for adaptive fields that they can assume from lower breakpoints. This would be enough:
```suggestion
    gridColumn: '1 / -1'
```

Reply:
fixed